app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    person_list = [Person(entity["name"], entity["age"]) for entity in people]
+
+    for entity in people:
```

Comment:
person will be a better name
```suggestion
    for person in people:
```

Reply:
change it all of the code